docs/troubleshooting.md

```diff
@@ -12,7 +12,8 @@ Table of contents
 1. [No packages show up in the UI (No OperatorHub Items Found)](#no-packages-show-up-in-the-ui-(no-operatorhub-items-found))
 2. [OperatorSource failing to download from datastore](#operatorSource-failing-to-download-from-datastore)
 3. [OperatorSource `MESSAGE` reads unknown error (status 404)](#operatorsource-`message`-reads-unknown-error-(status-404)) 
-4. [CatalogSourceConfig stuck in `Configuring` phase](#catalogsourceconfig-stuck-in-configuring-phase)
+4. [Operator/s in an OperatorSource fail to show up in the UI](#operator/s-in-an-operatorSource-fail-to-show-up-in-the-ui)
```

Comment:
Why the `/s`? If you are trying to account for singular and plural it should be `(s)`.